Packs/NozomiNetworks/Integrations/NozomiNetworks/NozomiNetworks.py

```diff
@@ -218,7 +223,8 @@ def ack_unack_alerts(ids, status, client):
     data = []
     for id in ids:
         data.append({'id': id, 'ack': status})
-    client.http_post_request('/api/open/alerts/ack', {'data': data})
+    if data:
```

Comment:
What was happening before? Was there an error occurring when there was no data?

Reply:
There was no errors, but we send a request with an empty body.
This kind of request is useless because does not change the status of the called environment, so it is a little performance improvement to not do the call if there are not incidents to ack.